egs2/wsj_kinect/enh1/local/data.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# Author: Atharva Anand Joshi (atharvaa@andrew.cmu.edu)
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+help_message=$(cat << EOF
+Usage: $0
+EOF
+)
+
+. ./db.sh
+. ./path.sh
+
+### This is a parameter to allow parallel execution of the script.
+### It reduces the execution time from ~12 hours to ~1.5 hours
+### Number of threads - tr: 50, cv: 13, tt: 8
+parallel=true
+
+### This parameter is used to decide whether to use dereverb or reverb references during training.
+use_dereverb=false
+
+### The following two parameters are currently fixed for Kinect-WSJ
+#min_or_max=min
+#sample_rate=16k
+
+output_path=$PWD/data/2speakers_reverb_kinect
+tr="tr"
+cv="cv"
+tt="tt"
+pdir=$PWD
+spk1_dir="s1"
+spk2_dir="s2"
+
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+if [ ! -e "${WSJ0_2MIX}" ]; then
+    log "Fill the value of 'WSJ0_2MIX' of db.sh"
+    exit 1
+fi
+if [ ! -e "${CHIME5}" ]; then
+    log "Fill the value of 'CHIME5' of db.sh"
+    exit 1
+fi
+if [ ! -e "${DIHARD2}" ]; then
+    log "Fill the value of 'DIHARD2' of db.sh"
+    exit 1
+fi
+
+if [ "${use_dereverb}" == true ]
+then
+    echo "Using dereverb references"
+    spk1_dir="s1_direct"
+    spk2_dir="s2_direct"
+fi
+
+
+### Download the scripts for noise extraction and mixture creation
+echo "Downloading scripts for wsj_kinect"
+url=https://github.com/sunits/Reverberated_WSJ_2MIX/archive/refs/heads/master.zip
+datadir=data
+wdir=data/scripts
+
+if [ -d $wdir ]; then
+  echo "Replacing $wdir"
+  rm -r $wdir
+fi
+
+for x in ${tr} ${cv} ${tt};
+do
+  if [ -d $datadir/$x ]; then
+  echo "Clearing $x"
+  rm -r ${datadir:?}/${x:?}
+  fi
+done
+
+mkdir -p $wdir
+
+wget --continue -O $wdir/mixture_scripts.zip ${url}
+unzip $wdir/mixture_scripts.zip -d $wdir
```

Comment:
How about just doing `git clone ...` to fetch the repository into `$wdir`?

Reply:
Replaced `wget` with `git clone` in data.sh